tests/test_tools.py

```diff
@@ -491,6 +492,41 @@ def test_is_dynamic_cwd_width():
         obs = is_dynamic_cwd_width(inp)
         yield assert_equal, exp, obs
 
+def test_is_tb_opt():
```

Comment:
Ahh. Sorry. Just saw it was tests of the individual functions with the same name. My bad :)